test/e2e/cloudstack_test.go

```diff
@@ -1753,15 +1753,42 @@ func TestCloudStackKubernetes123RedHatAPI(t *testing.T) {
 	test := framework.NewClusterE2ETest(
 		t, cloudstack, framework.WithEnvVar(features.FullLifecycleAPIEnvVar, "true"),
 	).WithClusterConfig(
-		cloudStackAPIClusterBaseChanges(cloudstack),
```

Comment:
Instead of using this, the clusters are configured directly in the test. It's easier to follow the test this way.